spec/acceptance/init_spec.rb

```diff
@@ -21,5 +21,15 @@
     describe process('puppet-summary') do
       its(:args) { is_expected.to match %r{\ -host\ 127.0.0.1\ -port\ 4321} }
     end
+    # see https://github.com/skx/puppet-summary/blob/master/API.md
+    it 'works on get calls' do
+      shell('curl -s http://127.0.0.1:4321/?accept=application/json') do |r|
```

Comment:
is curl guaranteed to be on the path?

Reply:
yep. somewhere deep down in the nodeset the curl system package is installed and beaker puts almost every existing directory into the $PATH it gets from the running container, it has to be there somewhere. We use the same test on other modules. In the past I tried an absolute path to be 100% sure, but of course that's different on every OS :(
tl;dr this is fine